import-export-cli/cmd/k8s/genDeploymentDir.go

```diff
@@ -0,0 +1,277 @@
+				utils.HandleErrorAndExit("Cannot copy metadata file from the source directory ", err)
+			}
+			fmt.Println(fileName)
+			metaDataYamlFile, err := ioutil.ReadFile(filepath.Join(sourceDirectoryPath, fileName))
+			if err != nil {
+				utils.HandleErrorAndExit("Cannot read the meta file", err)
+			}
+			errUnmarshal := yaml.Unmarshal(metaDataYamlFile, &apiMetaData)
+			if errUnmarshal != nil {
+				utils.HandleErrorAndExit("Error unmarshal api configmap into struct ", errUnmarshal)
+			}
+			apiCrd.Name = apiMetaData.Name
+			apiCrd.Spec.SwaggerConfigMapName = fmt.Sprintf("%v-cm", apiMetaData.Name)
+			apiCrd.Spec.ParamsValues = fmt.Sprintf("%v-params", apiMetaData.Name)
+			apiCrd.Spec.CertsValues = fmt.Sprintf("%v-certs", apiMetaData.Name)
+
+			break
+		} else if strings.EqualFold(fileName, utils.MetaFileApplication) { // if project artifact is a Application project
+			metaDataFileFound = true
+			err := utils.CopyFile(filepath.Join(sourceDirectoryPath, fileName), filepath.Join(deploymentDirPath, utils.MetaFileApplication))
+			if err != nil {
+				utils.HandleErrorAndExit("Cannot copy metadata file from the source directory ", err)
+			}
+			metaDataYamlFile, err := ioutil.ReadFile(deploymentDirPath + fileName)
+			if err != nil {
+				utils.HandleErrorAndExit("Cannot read the meta file", err)
+			}
+			errUnmarshal := yaml.Unmarshal(metaDataYamlFile, &apiMetaData)
+			if errUnmarshal != nil {
+				utils.HandleErrorAndExit("Error unmarshal api configmap into struct ", errUnmarshal)
+			}
+			apiCrd.Name = apiMetaData.Name
+			apiCrd.Spec.SwaggerConfigMapName = fmt.Sprintf("%v-cm", apiMetaData.Name)
+			apiCrd.Spec.ParamsValues = fmt.Sprintf("%v-params", apiMetaData.Name)
+			apiCrd.Spec.CertsValues = fmt.Sprintf("%v-certs", apiMetaData.Name)
+			break
+		}
+	}
+	// if *_meta.yaml is not found inside the source directory
+	if !metaDataFileFound {
+		utils.HandleErrorAndExit("Cannot find metadata file inside the source directory ", err)
+	}
+
+	// write to api_crd.yaml file
+	byteVal, errMarshal := yaml.Marshal(apiCrd)
+	if errMarshal != nil {
+		utils.HandleErrorAndExit("Error marshal api configmap ", errMarshal)
+	}
+	err = ioutil.WriteFile(filepath.Join(deploymentDirPath, "api_crd.yaml"), byteVal, os.ModePerm)
+	if err != nil {
+		utils.HandleErrorAndExit("Error creating api_crd.yaml file", err)
+	}
+
+	apiParamsData, _ := box.Get("/sample/api_params.yaml")
+	apiParamsCmData, _ := box.Get("/kubernetes_resources/api_params_cm.yaml")
+	apiParamsCm := &corev1.ConfigMap{}
+	errUnmarshal = yaml.Unmarshal(apiParamsCmData, apiParamsCm)
+	if errUnmarshal != nil {
+		utils.HandleErrorAndExit("Error unmarshal api configmap into struct ", errUnmarshal)
+	}
+	apiParamsCm.Name = apiCrd.Spec.ParamsValues
+	apiParamsCm.Data = map[string]string{"params.yaml": string(apiParamsData)}
+	byteParamsVal, errParamsMarshal := yaml.Marshal(apiParamsCm)
+	if errParamsMarshal != nil {
+		utils.HandleErrorAndExit("Error marshal api configmap ", errMarshal)
```

Comment:
```suggestion
		utils.HandleErrorAndExit("Error marshal API configmap ", errMarshal)
```
Shall we use API instead of "api"? Please check all the applicable places (especially inside error messages and examples)

Reply:
Fixed with aa03cd8b4d47fd7fec7dc79a709f5a9f8cba9fd1